tests/Shrink/Extra.elm

```diff
@@ -28,6 +29,7 @@ The `LazyList a` type used by shrinkers is not exposed outside `elm-explorations
     map3 : (a -> b -> c -> d) -> LazyList a -> LazyList b -> LazyList c -> LazyList d
 
 -}
+map3 : (a -> b -> c -> d) -> LazyList a -> LazyList b -> LazyList c -> LazyList d
```

Comment:
Hi @harrysarson! Where does this `LazyList` get imported from?

Reply:
ah, very good point!

Maybe is a bug in the language server type inference

Reply:
Is just annoying because now I get squigly lines in my IDE and I cannot work out how to turn them off!